docs/en/Running-Hadoop-MapReduce-on-Alluxio.md

```diff
@@ -14,56 +14,73 @@ run your MapReduce programs with files stored on Alluxio.
 
 ## Initial Setup
 
-The prerequisite for this part is that you have [Java](Java-Setup.html). We also assume that you
-have set up Alluxio and Hadoop in accordance to these guides
+The prerequisite for this guide includes
+
+- You have [Java](Java-Setup.html). 
+- You have set up an Alluxio cluster in accordance to these guides
 [Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
-In order to run some simple map-reduce examples, we also recommend you download the
+- In order to run some simple map-reduce examples, we also recommend you download the
 [map-reduce examples jar](http://mvnrepository.com/artifact/org.apache.hadoop/hadoop-mapreduce-examples/2.4.1) based on your hadoop version,
```

Comment:
Maybe the link should point to a version-less link? http://mvnrepository.com/artifact/org.apache.hadoop/hadoop-mapreduce-examples